package.json

```diff
@@ -3,13 +3,13 @@
   "version": "4.9.8",
   "description": "Dark theme for every website",
   "scripts": {
-    "api": "node -e \"require('./tasks/bundle-api')()\"",
+    "api": "node tasks/build.js --api",
     "benchmark-server": "node tests/benchmark-server/index.js",
-    "build": "node tasks/release.js",
+    "build": "node tasks/build.js --build",
```

Comment:
The `build` script is confusing. The `api` script is also build, `release` is build, and `debug` is also build. Since `build` is used for debugging, I suggest you rename it into `debug`, and rename `debug` into `debug-watch`.